aten/src/ATen/native/native_functions.yaml

```diff
@@ -9006,6 +9006,19 @@
   python_module: linalg
   variants: function
 
+- func: linalg_vector_norm(Tensor self, Scalar? ord=None, int[1]? dim=None, bool keepdim=False, *, ScalarType? dtype=None) -> Tensor
+  python_module: linalg
+  variants: function
+  dispatch:
+    DefaultBackend: linalg_vector_norm
+
+- func: linalg_vector_norm.out(Tensor self, Scalar? ord=None, int[1]? dim=None, bool keepdim=False, *, ScalarType? dtype=None, Tensor(a!) out) -> Tensor(a!)
+  use_c10_dispatcher: hacky_wrapper_for_legacy_signatures
```

Comment:
Can this ditch the hacky wrapper?

Reply:
Looks like every `linalg_*.out` function except for `linalg_slogdet.out` uses this. If I remove the wrapper, I get a linker error. I'm not sure what's different about `linalg_slogdet.out`, I'll see if I can find out.

Reply:
I figured it out. Evidently, if I don't use `hacky_wrapper_for_legacy_signatures`, the `at::*_out` function needs to be declared with `result` as the last argument: https://github.com/pytorch/pytorch/blob/a51b9a823cff20d5a0bcd299e11e5cf542814640/aten/src/ATen/native/README.md#use_c10_dispatcher

Reply:
I suppose all linalg functions should be updated to avoid `hacky_wrapper_for_legacy_signatures` eventually, right?